.env.example

```diff
@@ -0,0 +1,15 @@
+# Django secret key used on app settings
+SECRET_KEY="your_secret_here"
+
+# Authentication for koiki API.
+KOIKI_HOST="https://rekistest.koiki.es/services"
+KOIKI_AUTH_TOKEN="your_auth_token here"
+
+# Wordpress woocommerce auth
+WCFMMP_HOST="your_wp_woocomerce_url"
+WCFMMP_USER="your_wp_woocomerce_user"
+WCFMMP_PASSWORD="your_wp_woocomerce_password"
+
+# Details to connect to WCFM, the Marketplace plugin's API on Woocommerce.
```

Comment:
You mixed this one and the comment above. This block refers to Woocommerce's webhooks while the previous refers to the marketplace plugin.

Reply:
You're right.
Fixed by: https://github.com/coopdevs/lazona_connector/pull/92/commits/367a2d2a55d752d8b07cc036e37c65c51815d06e